### ISSUE TITLE
Script fails to run when spawning lxterminal due to invalid arguments

### ISSUE THREAD

**Issue by khalim19** [megastep/makeself#134]
On Lubuntu, when executing the makeself-created script in the file explorer GUI, a dialog prompts the user to execute the script using either the option "Execute" or "Execute in Terminal". The latter works fine as the desktop environment (LXDE) seems to spawn a terminal instance on its own, but the former fails to run because the script tries to run the `lxterminal` command with `-title` and `--xwin` arguments which do not exist in `lxterminal`.

There may be more terminal commands which may not accept the two aforementioned arguments (e.g. `gnome-terminal` or `xfce4-terminal` accept `--title` instead of `-title` according to their man pages).

**Comment by megastep** [comment]
Yeah, the problem is that right now Makeself searches for various known commands for a terminal, and assumes each of these take the same basic arguments. So it looks like it fails in some cases, and we should probably handle this case better.

**Comment by 00sapo** [comment]
Not sure that this is linked:

when you try to run a makeself zip as subprocess, it breaks with 

> # Failed to parse arguments: Unknown option -title

Tested with Python 3.6 and QML.

Reproducible steps:
* create a makeself zip running something (e.g. a shell Hello World)
* open python terminal and run:
```
import os
print(os.popen("./zipapp --").read())
```
Similar for QML or other languages.

**Comment by psy0rz** [comment]
I can confirm this is also a problem with gnome-terminal, -title is not supported.
